src/main/java/org/gelecekbilimde/scienceplatform/post/controller/CategoryController.java

```diff
@@ -25,16 +23,35 @@ class CategoryController {
 
 	private final CategoryToResponseMapper categoryToResponseMapper = CategoryToResponseMapper.initialize();
 
-	@GetMapping
+	@GetMapping("/categories")
```

Comment:
Sanırım burası summary yani özet servisi olacaktı. Bir tane ekstra olarak yönetim panelinde göstereceğimiz listeleme servisine ihtiyacımız var. Orada pagination filter gibi özellikler de olacaktı.
```suggestion
	@GetMapping("/categories/summary")
```